### ISSUE TITLE
NVDA's reporting of the remaining time is broken in Foobar2000

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#12197]
Control+SHIFT+R reports the remaining time in Foobar2000, but now it doesn't work and NVDA remains silent.
### Steps to reproduce:
1. Run Foobar2000 and play a track.
2. Press Control+SHIFT+R.
### Actual behavior:
NVDA remains silent. Note that Control+SHIFT+E and Control+SHIFT+T still work.
### Expected behavior:
NVDA should report the remaining time.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
NVDA Alpha 21882, but many alphas before it also exhibit the issue.
#### Windows version:
Win 10 Pro 64-Bit, Version 20H2 (OS Build 19042.867)
#### Name and version of other software in use when reproducing the issue:
foobar2000 version 1.6.2.0
#### Other information about your system:
I've even reset all Foobar 2000 settings, but the issue persists.
#### Does the issue still occur after restarting your computer?
Yes
#### If add-ons are disabled, is your problem still occurring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by nidza07** [comment]
I've just yesterday noticed this issue on my end as well, but wanted to investigate if it was my Foobar. Thanks @amirsol81 for investigating, I can reproduce it here as well.

**Comment by nidza07** [comment]
Here is a log demonstrating the issue:
[Foobar remaining time bug.log](https://github.com/nvaccess/nvda/files/6167680/Foobar.remaining.time.bug.log)

**Comment by josephsl** [comment]
Hi, based on the log, it appears this is something to do with wxPython (again…) + Python 3.8. Thanks.

**Comment by nidza07** [comment]
Any updates  on  this one? It can still be reproduced using alpha-22050,83738d7d

**Comment by amirsol81** [comment]
@seanbudd This is still reproduceable with Alpha 22160. Is it supposed to fix the Foobar2000 issue?

**Comment by seanbudd** [comment]
@amirsol81, no, the alpha snapshot seems to not be published yet due to a build failure. I've queued the alpha/master branch to rebuild

**Comment by seanbudd** [comment]
@amirsol81, alpha-22176 , the alpha snapshot which incorporates these changes has been published now

**Comment by amirsol81** [comment]
@seanbudd Thanks. Alpha 22176 fixes Foobar2000's issue.
